queries.sql

```diff
@@ -129,3 +129,94 @@ LEFT JOIN animals ON owners.id = animals.owner_id
 GROUP BY owners.full_name
 ORDER BY animal_count DESC
 LIMIT 1;
+
+/* Query Questions */
+
+-- Who was the last animal seen by William Tatcher?
+
+SELECT a.name AS william_last
+FROM visits v
+LEFT JOIN vets vt ON v.vet_id = vt.id
+LEFT JOIN animals a ON v.animal_id = a.id
+WHERE vt.name = 'William Tatcher'
+ORDER BY v.date_of_visit DESC
+LIMIT 1;
```

Comment:
Well done updating your queries as requested by the previous reviewer. 👍 

However, to verify we have run the queries with success. it would be advisable to include screenshots of the updated query results in the pull request description, showcasing the modifications made. 🛠 🤗